### ISSUE TITLE
Texinfo not working

### ISSUE THREAD

**Issue by ocaisa** [hpcugent/easybuild-easyconfigs#1839]
The texinfo easyconfig should be removed since it doesn't actually work. It has a dependency on tex that is not present. I doubt we're going to port tex to eb and without doing so the final step of a proper installation:
```
make TEXMF=/your/texmf install-tex
```
can't be done since this would require write permissions into an OS dir.

**Comment by boegel** [comment]
How about adding `tex` as an OS dep, rather than removing the easyconfig?

This easyconfig passes the regtest, so it works for us (tm).

Or are you saying the installation process being performed is incomplete, thus not showing the brokenness?

**Comment by ocaisa** [comment]
I think whether it works or not is very dependent on your OS setup.
```
Installation notes specific to Texinfo:

* texinfo.tex and the other *.tex files are not installed by `make
  install'.  This is because there is no reliable way to know where to
  install them.  Instead, you have to run an additional make command
  after the normal make install:
    make TEXMF=/your/texmf install-tex
  texinfo.tex is installed in ${TEXMF}/tex/texinfo/texinfo.tex, etc.

```
For a pretty clean CentOS 7 frontend it does not work correctly since no texinfo.tex file is created (or added to the right place) so the installation is incomplete. For me, this manifested itself when I tried to build the documentation for Octave.

**Comment by ocaisa** [comment]
Ok, I made a pull request that should fix this #1840

**Comment by boegel** [comment]
fixed with #1840
